packages/block-editor/src/store/actions.js

```diff
@@ -218,6 +218,29 @@ export function toggleSelection( isSelectionEnabled = true ) {
 	};
 }
 
+function getBlocksWithAutoApplyStyles( blocks, blockEditorSettings ) {
```

Comment:
Since we neatly separate the _getting_ of `blockEditorSettings` from the logic of `getBlocksWithAutoApplyStyles`, we should add a couple of unit tests for this, as well as add some JSDoc for the function.